repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/services/opentracing/OpenTracingServiceDisabledHttpTest.groovy

```diff
@@ -1,152 +0,0 @@
-/*
- * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
- * Repose
- * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
- * Copyright (C) 2010 - 2015 Rackspace US, Inc.
- * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
- */
-package features.services.opentracing
-
-import org.openrepose.framework.test.ReposeValveTest
-import org.openrepose.framework.test.mocks.MockTracerAgent
-import org.openrepose.framework.test.mocks.MockTracerCollector
-import org.rackspace.deproxy.Deproxy
-import spock.lang.Unroll
-
-class OpenTracingServiceDisabledHttpTest extends ReposeValveTest {
```

Comment:
Can we keep this test? Instead of using the `enabled` attribute, you can just not include an `open-tracing.cfg.xml` file. The assertions should be good, though.

Reply:
There's already a test where there's no config.

Reply:
https://github.com/rackerlabs/repose/pull/1892/files#diff-22becc3deae6d86316e46362feff7c24